ILogger_best_practice/kb/scenario_description/defaults.md

```diff
@@ -0,0 +1,182 @@
+# Defaults
```

Comment:
Overall, all the info here seemed very accurate. If intended for a human to read the broader structure of the document seemed disjoint though. First there is some info about string interpolation and then some info about source-generated logging but nothing that ties those two parts together.